docs/05_operations/11_tracing/01_tracing.md

```diff
@@ -0,0 +1,90 @@
+---
+title: 'Tracing'
+sidebar_label: 'Tracing'
+id: tracing
+keywords: [operations, tracing]
+tags:
+    - operations
+    - tracing
+---
+
+The Genesis Tracing module enables you to capture tracing for your application through its integration with [OpenTelemetry](https://opentelemetry.io/). OpenTelemetry is a vendor-neutral open-source observability freamework and has become an industry-standard.
```

Comment:
I'm guessing you meant framework not freamework

Reply:
Yep, thanks!